test/box.purchase.test.ts

```diff
@@ -4,6 +4,7 @@ import { MockProvider, solidity } from 'ethereum-waffle'
 import { ethers } from 'hardhat'
 
 import { bondingCurve as bc } from './helpers'
+import { getAddress } from '@ethersproject/address'
```

Comment:
Not being used

Reply:
yeah i've been leaving things like this since i end up re adding them to build tests and then deleting the lines.  im not overly concerned about unused imports in tests